packages/salesforcedx-vscode-apex/package.nls.json

```
@@ -8,6 +8,7 @@
   "show_error_title": "Display Error",
   "go_to_definition_title": "Go to Definition",
   "java_home_description": "Specifies the folder path to the Java 8 or Java 11 runtime used to launch the Apex Language Server (for example, /Library/Java/JavaVirtualMachines/jdk1.8.0_131.jdk/Contents/Home).",
+  "java_memory_description": "Amount of memory to allocate to Java, in MB, or null to use system default",
```

```suggestion
  "java_memory_description": "Amount of memory to allocate to the Apex Language Server, in MB, or null to use system default",
```
Do you think we should be explicit about it being for the Apex language server process?